01-Login/Auth0Sample/HomeViewController.m

```diff
@@ -53,15 +53,11 @@ - (IBAction)showLoginController:(id)sender {
                       }];
         
     }else{
-        [auth logOutUserWithCallback:^(BOOL response, NSString * _Nonnull message) {
+        [auth logOutUserWithCallback:^(BOOL response) {
             if(response){
-                [self showAlertWithMessage:[NSString stringWithFormat:@"%@", message]];
                 logged = false;
                 [sender setTitle:@"Sign in" forState:UIControlStateNormal];
             }
-            else {
```

Comment:
Looks better, add a little failure message (even if unlikely to happen in practice)